dGame/dGameMessages/GameMessages.h

```diff
@@ -833,6 +833,15 @@ namespace GameMessages {
 	struct ResetModelToDefaults : public GameMsg {
 		ResetModelToDefaults() : GameMsg(MessageType::Game::RESET_MODEL_TO_DEFAULTS) {}
 	};
-};
 
+	struct EmotePlayed : public GameMsg {
+		EmotePlayed() : GameMsg(MessageType::Game::EMOTE_PLAYED), emoteID(0), targetID(0) {}
```

Comment:
In the future don't chain constructors for a zero initialization, instead prefer using `{}` on the member itself like int32_t emoteID{};